samcli/commands/init/__init__.py

```diff
@@ -3,6 +3,8 @@
 Init command to scaffold a project app from a template
 """
 import logging
+import json
+from json import JSONDecodeError
```

Comment:
I'd recommend not having both a full and a partial import of `json` - you can use `json.JSONDecodeError` below, or import any json methods you use as a part of the `from` statement.

Reply:
I'd probably go with the former approach since `JSONDecodeError` is only used once.